crates/bevy_ui/src/lib.rs

```diff
@@ -123,7 +123,9 @@ impl Plugin for UiPlugin {
             .add_systems(
                 PreUpdate,
                 ui_focus_system.in_set(UiSystem::Focus).after(InputSystem),
-            );
+            )
+            .add_event::<Click>()
+            .add_systems(Update, ui_click.after(ui_focus_system));
```

Comment:
If you add this system to `Update`, then you don't need to use `after` because `ui_focus_system` runs in `PreUpdate`.
But you probably want to put it to `PreUpdate`.